src/main/java/com/hotline/pageobject/pages/RegistrationPage.java

```diff
@@ -0,0 +1,53 @@
+package com.hotline.pageobject.pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import java.util.List;
+
+public class RegistrationPage extends AuthorizationPage {
+    private WebElement nameField;
+    private WebElement registerBtn;
+    private WebElement showPasswordBtn;
+    private WebElement errorMsg;
+
+    public RegistrationPage (WebDriver driver) {
+        super(driver);
+        nameField = driver.findElement(By.name("name"));
+        registerBtn = driver.findElement(By.id("submit-button"));
+        showPasswordBtn = driver.findElement(By.cssSelector(".check-view"));
+    }
+
+    public WebElement getNameField() {
+        return nameField;
+    }
+
+    public WebElement getRegisterBtn() {
+        return registerBtn;
+    }
+
+    public WebElement getShowPasswordBtn() {
+        return showPasswordBtn;
+    }
+
+    public RegistrationFinalPage gotoRegistrationFinalPage() {
+        return new RegistrationFinalPage(driver);
+    }
+
+    public WebElement getErrorMsg() {
+        return driver.findElement(By.cssSelector(".errors"));
```

Comment:
What if there are multiple elements with chosen locator and this method returns different from the one you're expecting?

Reply:
I thought there was some reason but it seems tests are running well with it in constructor)

Reply:
@andyyarych I would create than specific webElement for each after field possible error